src/cljs/athenaeum/views.cljs

```diff
@@ -0,0 +1,11 @@
+(ns athenaeum.views)
+
+(defn home-page
+  []
+  [:div
+   [:p "hello world"]])
+
+(defn page-not-found
+  []
+  [:div
+   [:p "Page not found."]])
```

Comment:
Have you checked that hot reloading works? You have to add a little more code here to get hot reloading to work properly. https://code.thheller.com/blog/shadow-cljs/2019/08/25/hot-reload-in-clojurescript.html

Try changing the home page text in the source code, and see if it changes in the browser.